multierror.go

```diff
@@ -49,3 +50,7 @@ func (e *Error) GoString() string {
 func (e *Error) WrappedErrors() []error {
 	return e.Errors
 }
+
+func (e *Error) MarshalJSON() ([]byte, error) {
+	return json.Marshal(e.Errors)
```

Comment:
@lukaszatsagan :+1: 

WRT 
> do we want to do it in the library at all

@malpiatko and I had a discussion about whether it's going to be more appropriate to wrap `multi.Error` in supernova. While I think it's marginally cleaner, it's waaaay more difficult because of the `field, not a method` error